projects/pastanaga/src/lib/dialog/README.md

```diff
@@ -338,3 +338,7 @@ export class SomeDialogComponent implements IDialog {
 }
 ```
 
+## Refresh
+After changing some values dynamically, we might want to have those changes reflected.
+
+That can be done using `refresh`.
```

Comment:
Not very clear.

When you change dialog properties from outside the dialog, like
```typescript
this.dialog.displayBackButton = true 
```
you'll need a `markForCheck` but from the dialog itself, not from the dialog wrapper.

`refresh` method is there in this purpose.